numpy/core/numeric.py

```diff
@@ -1928,6 +1928,10 @@ def fromfunction(function, shape, **kwargs):
     return function(*args, **kwargs)
 
 
+def _frombuffer(buffer, dtype, shape):
+    return frombuffer(buffer, dtype=dtype).reshape(shape)
```

Comment:
Maybe it would be possible to make `numpy.frombuffer` directly accept an optional `shape` argument?